fritz.defaults.yaml

```diff
@@ -17,18 +17,34 @@ app:
       exact: True
     - path: "/source/:id"
       component: Source
+    - path: "/candidate/:id"
+      component: Candidate
     - path: "/groups"
       component: Groups
     - path: "/group/:id"
       component: Group
     - path: "/profile"
       component: Profile
+    - path: "/candidates"
+      component: CandidateList
     - path: "/sources"
       component: SourceList
     - path: "/user/:id"
       component: UserInfo
-
-  kowalski_token:
+    - path: "/upload_photometry/:id"
+      component: UploadPhotometry
+    - path: "/about"
+      component: About
+#    - path: "/alerts"
+#      component: Alerts
+    - path: "/alerts/ztf/:id"
+      component: ZTFAlert
```

Comment:
Again, is it necessary that `Alert`s from different instruments live in different tables and have different endpoints? Since `Alert`s are unstructured JSON, I think it would be fine to have them all live in a single table with an index on `instrument_id`.

Reply:
They don't live in SP tables -- SP just makes K API calls and renders the responses. Alerts would come from different collections in K's mongo so the calls would be different. But it's fine, we can figure out what to query by the object name (see above) -- will implement this.

Reply:
I would actually prefer keeping this as is: the logic for rendering alerts from different streams will likely differ significantly, so I don't want to dump everything into a single component.